app/design/adminhtml/Magento/backend/web/css/source/forms/_fields.less

```diff
@@ -127,6 +127,28 @@
             #mix-grid .column(@field-label-grid__column, @field-grid__columns);
         }
 
+        &.field-subscription {
```

Comment:
I think you misunderstood me. That element specific design should be avoided and template/component should be changed instead so it could render correct template that looks better.